### DISCUSSION TITLE
Unable to move through options when screenreader is turned on

### DISCUSSION THREAD

**Discussion by SairamAlagapan** [JedWatson/react-select#5636] [Q&A]
I tested react-select home page in my iPhone device with Voiceover turned on and I was able to move through options using "Swipe right" but, it does not work in my local application. Do I need to pass any specific property for this to work?

**Comment by souvik666** [comment]
hey @SairamAlagapan  it might help [link](https://stackoverflow.com/questions/74888982/cant-access-select-on-keyboard-with-screen-reader)
